src/06_tuples.py

```diff
@@ -17,28 +17,33 @@
 
 import math
 
+
 def dist(a, b):
     """Compute the distance between two x,y points."""
     x0, y0 = a  # Destructuring assignment
     x1, y1 = b
 
     return math.sqrt((x1 - x0)**2 + (y1 - y0)**2)
 
+
 a = (2, 7)   # <-- x,y coordinates stored in tuples
 b = (-14, 72)
 
 # Prints "Distance is 66.94"
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
+t = (1, 2, 5, 7, 99)
+
+
+def print_tuple(test):
+    for i in test:
+        print(i)
 
-# YOUR CODE HERE
 
-t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
 # Declare a tuple of 1 element then print it
 u = (1)  # What needs to be added to make this work?
-print_tuple(u)
+print_tuple(str(u))
```

Comment:
This will print out each character in the value u (which as there's only one can make it seem like this is working). Remembering that a tuple is a comma separated set of values, the way to make "u" a tuple is to put a comma after the "1". Then you don't need to convert it to a str.